src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -58,8 +58,7 @@
     ElasticPoolCapabilitiesAdditionalDetails,
     FailoverPolicyType,
     SqlServerMinimalTlsVersionType,
-    SqlManagedInstanceMinimalTlsVersionType,
-    AuthenticationType
```

Comment:
AuthenticationType is removed from SDK. This shouldn't be part of my change but I was getting build errors. I think I didn't broke anything 🤞